### ISSUE TITLE
Issue plotting XZ plane with PlotEqSlice

### ISSUE THREAD

**Issue by wesleyjmartin** [mattkjames7/PyGCPM#1]
Hello, 

I am using the [PyGCPM](https://github.com/mattkjames7/PyGCPM/tree/master) wrapper to reproduce plots from the [original model](https://plasmasphere.nasa.gov/models/). I've successfully reproduced the [XY](https://plasmasphere.nasa.gov/media/kp.html) plots using the PlotEqSlice function, but I've encountered some challenges when attempting to reproduce the [XZ meridian swing](https://plasmasphere.nasa.gov/media/meridianswing.html) plots. While experimenting with the code and attempting to swap Y and Z values, I noticed that the GCPM function returns nan when x^2 + y^2 < z^2. This causes most of the plot to be empty when I attempt to use XZ coordinates. How can this code be used to reproduce the meridian swing plots in the XZ plane?

**Comment by mattkjames7** [comment]
Hi, 
Thanks for using my code. I must admit - I had not tested this function very much (obviously). It turns out that there was a coordinate conversion problem. I appear to have fixed it, e.g.:
```python
import PyGCPM
PyGCPM.PlotMLTSlice(12.0,20010101,12.0,scale=[0.01,1e6],cmap='jet',dR=0.1,Verbose=True)
```
![Screenshot from 2023-11-20 23-06-12](https://github.com/mattkjames7/PyGCPM/assets/17782733/240922bd-ea9d-413f-bf34-1e2918e6e165)

I have updated the code here on git, I have also uploaded a new package to PyPI. The new version is 0.1.0 and can be installed with `pip install PyGCPM`. The previous version would generate files when it compiled itself - it might be necessary to manually remove any files in site-packages.

Hope that helps!

Cheers,
Matt.
